tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -0,0 +1,182 @@
+<tool id="tool_odv" name="ODV collection manager" version="@VERSION@+galaxy@VERSION_SUFFIX@" profile="22.05" license="MIT">
+    <description>tool for qualificiation and validation</description>
+    <macros>
+        <token name="@VERSION@">0.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">pokapok/qcv-odv-tool:@VERSION@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        export HOME=\$PWD &&
+        mkdir -p /runtime/config &&
+        mkdir -p '/runtime/data-in-raw/' &&
+        mkdir -p '/runtime/data-in-ref/' &&
+        mkdir -p '/runtime/data-out/' &&
+
+        cp '$input_raw' '/runtime/data-in-raw/${input_raw.element_identifier}' &&
+        #if $input_ref:            
+            cp '$input_ref' '/runtime/data-in-ref/${input_ref.element_identifier}' &&
+        #end if
+
+        #if $config.type == 'yes':
+            ## Define output YAML file
+            touch /runtime/config/config_file.yaml
+
+            ## Write YAML content
+            '/runtime/config/config_file.yaml' << echo"
```

Comment:
You can use `configfiles` for this. It would be much cleaner.

Reply:
Okay I'll see how to do that !

Reply:
hmm I've got an issue because my configfile is optionnal so it's not always needed. And when it is not needed the stuff to fill it in does not exist so obviously it errors.
And I can't put in the config file some `#if '${config.type} == "yes"` because then when I do need the config file it does not understand correctly the if in cheetah an cannoot parse it correctly ..

any thought ?

Reply:
done ! still learning a new English acronym with you ;) (in my humble opinion)